<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileExplorer</class>
 <widget class="QMainWindow" name="FileExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileExplorer</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdvanced">
    <property name="title">
     <string>Advanced</string>
    </property>
    <addaction name="actionCheck_Disk_Fragmentation"/>
    <addaction name="actionCheck_Security_Threats"/>
   </widget>
   <addaction name="menuAdvanced"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="treeDockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="dw_ext">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ownershipPermissionsChartDockWidget">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QDockWidget" name="informationDockWidget">
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <widget class="QDockWidget" name="chart_widget">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6"/>
  </widget>
  <action name="actionCheck_Disk_Fragmentation">
   <property name="text">
    <string>Check Disk Fragmentation</string>
   </property>
  </action>
  <action name="actionCheck_Security_Threats">
   <property name="text">
    <string>Check Security Threats</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
